TESTING-INSTRUCTIONS.md

```diff
@@ -9,13 +9,62 @@
 3. Create a new category called **Hoodie** with **Clothing** as the parent category in the **Product categories** on the right.
 4. Select **Clothing** and **Hoodie with Pocket** as well and click **Update**
 5. Create an order with a single item of **Hoodie with Pocket** (keep note of the total price)
-6. Run the action scheduler (make sure all are run)
+6. Run the action scheduler (make sure all are run), you can do this manually by going to **WooCommerce > Status > Scheduled Actions**
```

Comment:
Might specify the action name we need to run in case there are multiple in the queue.